src/Common/src/Interop/Windows/Winsock/Interop.GetAddrInfoExW.cs

```diff
@@ -30,7 +30,7 @@ internal static partial class Winsock
         );
 
         [DllImport("ws2_32.dll", ExactSpelling = true, SetLastError = true)]
```

Comment:
This should not need `SetLastError = true`. We do not check it, and the implementation does not set it either.

Reply:
Removed.